### PR TITLE
OS-2: Week 11: Fixed Hospital System and Medicine System

### PR DESCRIPTION
**Pull request by divyanshkhetan**
## Proposed changes
- Fixed stock system
- Fixed doctor's model
- Added Hospital Add view

## Types of changes

_Put an `x` in the boxes that apply_

-   [x] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] Other (please describe):

## Checklist

_Put an `x` in the boxes that apply_

-   [x] My code follows the [style guidelines of this project](https://docs.google.com/document/d/1GI2Hile8UbGZ82gFe1y4wAVPcNPXip1cmXTzog1S41U/edit)
-   [x] I have performed a self-review of my own code
-   [x] I have created new branch for this pull request
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have made corresponding changes to the documentation
-   [x] My changes generate no new warnings
-   [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots
image

### CODE REVIEW

**Comment by divyanshkhetan** [comment]
@akshatnema please look into this.

**Comment by NuthanReddy646** [comment]
Approve @akshatnema

**Review by akshatnema — COMMENTED** [review]
Review required from @PranshuNayak and @samay-rgb

File: FusionIIIT/applications/health_center/models.py [hunk 1]
```diff
@@ -60,11 +59,12 @@ class Medicine(models.Model):
...
     def __str__(self):
-        return self.medicine_id
+        return self.medicine_id.medicine_name
 
 class Hospital(models.Model):
     hospital_name=models.CharField(max_length=100)
-    phone=models.CharField(max_length=10)
+    hospital_address=models.CharField(max_length=255, default="")
+    hospital_phone=models.CharField(max_length=10, validators=[MinLengthValidator(10)], default="0000000000")
     def __str__(self):
```
Comment (PranshuNayak) [inline FusionIIIT/applications/health_center/models.py:68]: The field names need not be prefixed with hospital, address, and phone are self-explanatory, no need to use hospital_address and hospital_phone. Change hospital_name to name too.

**Review by PranshuNayak — CHANGES_REQUESTED** [review]
Please make sure to do all necessary changes at once while changing the model schema. In the previous PR, you added contact_no now you are removing it. If possible try to give all changes in a single migration file.

**Review by PranshuNayak — APPROVED** [review]

**Review by samay-rgb — APPROVED** [review]

**Pull request merged**